src/common/translation/i18n/en.json

```diff
@@ -22,6 +22,13 @@
     },
     "redirect": {
       "text": "Redirecting..."
+    },
+    "session": {
+      "expired": {
+        "message": "Your session have expired. <br>In order to continue with application <br> you will be redirect to homepage and start again. <br>All your current progress will be saved.",
```

Comment:
I think a simpler error message would be nice. Maybe just: "Your session has expired. Go to the front page."

This might also show up when people are finished registering and are signing up for events etc, so "Continue registration" is pretty confusing?

Reply:
Can you edit the sheet?